pulseapi/entries/serializers/curriculum.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+from ..models import Entry
+from .base import EntrySerializerWithCreators, EntryWithCreatorsBaseSerializer
+
+
+class CurriculumEntrySerializer(EntrySerializerWithCreators):
+    """
+    Serializes an entry based on the fact that we
+    know that it is a curriculum item.
+    
+    This has the same requirements as news items.
+    """
+
+    class Meta:
+        model = Entry
+        fields = EntryWithCreatorsBaseSerializer.Meta.fields
```

Comment:
I think we want `EntrySerializerWithCreators` for all Entry type models instead because it knows how to read/write creators. This means we probably don't need to override the `Meta` class here.